src/core/data.jl

```diff
@@ -467,43 +419,71 @@ function check_phases(data::Dict{String,Any})
     end
 end
 
+
+""
 function _check_phases(data::Dict{String,Any})
     if haskey(data, "phases") && data["phases"] < 1
         error("phase values must be positive integers, given $(data["phases"])")
     end
 end
 
+
 "checks that phase angle differences are within 90 deg., if not tightens"
 function check_voltage_angle_differences(data::Dict{String,Any}, default_pad = 1.0472)
     if InfrastructureModels.ismultinetwork(data)
         error("check_voltage_angle_differences does not yet support multinetwork data")
     end
```

Comment:
If we are generalizing this to multiphase data, should this check be removed?

Reply:
I think not, somebody could edit their data structure and make `data["phases"]=0` for example, which would break some code that relies on that key

Reply:
Go for it, it contributes to this issue, https://github.com/lanl-ansi/PowerModels.jl/issues/270